src/event/sys/windows/parse.rs

```diff
@@ -79,17 +79,21 @@ fn parse_key_event_record(key_event: &KeyEventRecord) -> Option<KeyEvent> {
             let character_raw = key_event.u_char;
 
             if character_raw < 255 {
+                // Invalid character
+                if character_raw == 0 {
+                    return None;
+                }
+
                 let mut character = character_raw as u8 as char;
 
                 if modifiers.contains(KeyModifiers::ALT) {
                     // If the ALT key is held down, pressing the A key produces ALT+A, which the system does not treat as a character at all, but rather as a system command.
                     // The pressed command is stored in `virtual_key_code`.
                     let command = key_event.virtual_key_code as u8 as char;
 
+                    // TODO: figure out if this is needed
```

Comment:
I tested it out, if I press the numbers + ALT the ascii number for all characters pressed along ALT is placed in the `character_raw`, this does not happen with Numpad numbers. 

The reason why there is an alphabetic check is that windows interpret ALT as a system key and will place the pressed character in `key_event.virtual_key_code`. This will only be done for alphabetic characters. However, as discovered above, we can just as well use `character` for this. 

Here we overwrite the 'character' with the system keycode which is only an alphabetic character. 
https://github.com/crossterm-rs/crossterm/pull/488/files#diff-ff9f93aac33f0c41012d6df50b082828R92

I propose to remove this line which will give us both numeric and alphabetic support. 

One major change is that the `virtual key code` is capitalized by default while `character` isn't. Perhaps we can call `to_upper`.

Reply:
I just realized this entire if statement line `89`-`97` are useless. We can remove this.

Reply:
I agree.